2022-04-05 05:39:38,029 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-04-05 05:39:38,030 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\gabdr\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\instabot\bot\bot.py
2022-04-05 05:39:38,030 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-04-05 05:39:38,134 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-04-05 05:39:38,135 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-04-05 05:39:38,135 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-04-05 05:39:38,136 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
2022-04-05 05:44:38,224 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-04-05 05:44:38,224 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-04-05 05:44:38,224 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-04-05 05:44:38,225 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 10 minutes.
